src/test/fuzz/script.cpp

```diff
@@ -119,4 +125,41 @@ void test_one_input(const std::vector<uint8_t>& buffer)
             wit.SetNull();
         }
     }
+
+    const char* op_name = GetOpName(ConsumeOpcodeType(fuzzed_data_provider));
+    assert(op_name != nullptr);
+
+    const char* error_string = ScriptErrorString(static_cast<ScriptError>(fuzzed_data_provider.ConsumeIntegralInRange<int>(0, SCRIPT_ERR_ERROR_COUNT)));
+    assert(error_string != nullptr);
+
+    {
+        const std::vector<uint8_t> bytes = ConsumeRandomLengthByteVector(fuzzed_data_provider);
+        CScript append_script{bytes.begin(), bytes.end()};
+        append_script << fuzzed_data_provider.ConsumeIntegral<int64_t>();
+        append_script << ConsumeOpcodeType(fuzzed_data_provider);
+        append_script << CScriptNum{fuzzed_data_provider.ConsumeIntegral<int64_t>()};
+        append_script << ConsumeRandomLengthByteVector(fuzzed_data_provider);
+    }
+
+    {
+        WitnessUnknown witness_unknown_1{};
+        witness_unknown_1.version = fuzzed_data_provider.ConsumeIntegral<int>();
+        const std::vector<uint8_t> witness_unknown_program_1 = fuzzed_data_provider.ConsumeBytes<uint8_t>(40);
+        witness_unknown_1.length = witness_unknown_program_1.size();
+        std::copy(witness_unknown_program_1.begin(), witness_unknown_program_1.end(), witness_unknown_1.program);
+
+        WitnessUnknown witness_unknown_2{};
+        witness_unknown_2.version = fuzzed_data_provider.ConsumeIntegral<int>();
+        const std::vector<uint8_t> witness_unknown_program_2 = fuzzed_data_provider.ConsumeBytes<uint8_t>(40);
+        witness_unknown_2.length = witness_unknown_program_2.size();
+        std::copy(witness_unknown_program_2.begin(), witness_unknown_program_2.end(), witness_unknown_2.program);
+
+        (void)(witness_unknown_1 == witness_unknown_2);
+        (void)(witness_unknown_1 < witness_unknown_2);
+
+        const CNoDestination no_destination_1;
+        const CNoDestination no_destination_2;
+        (void)(no_destination_1 == no_destination_2);
+        (void)(no_destination_1 < no_destination_2);
```

Comment:
this looks like a unit test that does not depend on any fuzz input

Reply:
My suggestion would be to construct a random `CTxDestination`.

Reply:
Done! :)